### PR TITLE
Update Plugin-Authoring Docs

### PR DESCRIPTION
**Pull request by MaZderMind**
When setting up a new Plugin according to the docs I found some small discrepancies and pieces of missing information. This PR adds or modifies the docs to how I found the steps to work.

## How Has This Been Tested?
Tested when setting up a new Plugin according to the docs

## Checklist
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My change is listed in the CHANGELOG.rst if appropriate.

### CODE REVIEW

File: doc/developer/plugins/customview.rst [hunk 1]
```diff
@@ -84,7 +87,7 @@ Frontend views
 --------------
 
 Frontend views work pretty much like organiser area views. Take care that your URL starts
-with ``/(P?P<event>[\]+)/``, or even better, ``/(P?P<event>[\]+)/p/``.
+with ``f'^(?P<event>[{SLUG_CHARS}]+)/``
```
Comment (rixx) [inline doc/developer/plugins/customview.rst:90]: We really want to encourage plugins to include `/p/` in their URL to reduce collisions, especially with future URLs, so I'd like to keep this part in (though with the proper `SLUG_CHARS` of course).

**Review by rixx — COMMENTED** [review]
Thank you! One small niggle, otherwise very happy to merge.

File: doc/developer/plugins/customview.rst (hunk 1 above)
Comment (MaZderMind) [reply doc/developer/plugins/customview.rst]: Fixed.

**Review by rixx — APPROVED** [review]
Thank you!

**Pull request merged**